composer.json

```diff
@@ -57,14 +57,15 @@
         "zetacomponents/persistent-object": "~1.8",
         "zetacomponents/signal-slot": "~1.2",
         "zetacomponents/system-information": "~1.1",
-        "zetacomponents/webdav": "~1.1"
+        "zetacomponents/webdav": "~1.1",
+        "ircmaxell/password-compat": "^1.0"
```

Comment:
hmm, as we now require php 5.5 as of cf005524485c89f82e6d24e979dfc26d659e45cd, we probably don't need this anymore.

Reply:
Right! Removed.